iris-mpc-gpu/src/server/actor.rs

```diff
@@ -757,6 +757,15 @@ impl ServerActor {
             }
         }
 
+        // Check for supermatchers for v1 compatibility and mark them as non-unique
+        const SUPERMATCH_THRESHOLD: usize = 4_000;
```

Comment:
yeah, but the config is not available here and would require to adapt the actor and a bunch of places where we call it, so I then thought not worth changing it, since we'll remove it anyways soon again.